u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java

```diff
@@ -135,14 +139,10 @@ public SecurityKeyData processRegistrationResponse(
     Log.info("-- Parsed rawRegistrationResponse --");
     Log.info("  userPublicKey: " + Hex.encodeHexString(userPublicKey));
     Log.info("  keyHandle: " + Hex.encodeHexString(keyHandle));
-    Log.info("  attestationCertificate: " + attestationCertificate.toString());
+    Log.info("  attestationCertificate (chain): " + attestationCertificateChain.toString());
```

Comment:
What does the " (chain)" buy you here, since you've asserted that the chain is exactly one certificate long?